tripgo.swagger.yaml

```diff
@@ -121,212 +131,180 @@ paths:
             required:
               - region
               - operator
             example:
               - region: US_CA_LosAngeles
+                operator: Metro - Los Angeles
                 modes:
                   - pt_pub_tram
                   - pt_pub_bus
-                operators:
-                  - Metro - Los Angeles
+                full: true
       responses:
         200:
           description: Successful response
           schema:
-            type: object
-            properties:
-              regionCode:
-                type: string
-                description: Internal identifier for the region.
-              operator:
-                type: string
-                description: TSP's short name.
-              numberOfServices:
-                type: integer
-                description: Total number of services for this TSP.
-              realTime:
-                type: object
-                $ref: '#/definitions/RealtimeSnapshot'
-              transitModes:
-                type: array
-                description: Detailed TSP information per public transit modes.
-                items:
-                  type: object
-                  properties:
-                    identifier:
-                      type: string
-                      description: A specific public transit mode-identifier, e.g., `pt_pub_bus` or `pt_pub_gondola`.
-                    numberOfServices:
-                      type: integer
-                      description: Total number of services for this TSP and mode.
-                    realTime:
-                      type: object
-                      $ref: '#/definitions/RealtimeSnapshot'
-                    routes:
-                      type: array
-                      description: Route information.
-                      items:
+            type: array
+            items:
+              type: object
+              description: Route information.
+              properties:
+                name:
+                  type: string
+                  description: Route name.
+                mode:
+                  type: array
+                  items:
+                    description: Detailed information for this route under a specific public transport mode. It could be the case that same route name is used by this operator under another mode.
+                    type: object
+                    properties:
+                      numberOfServices:
+                        type: integer
+                        description: Total number of services for this route and mode.
+                      stops:
+                        type: array
+                        items:
+                          type: string
+                          description: Stop code. DO WE USE STOP CODES TO REFER STOPS IN OUR ENDPOINTS?
```

Comment:
we use stop codes in latest.json, service.json, departures.json, etc...  they may be returned by locations.json, routing.json, waypoint.json.  maybe we can make a $ref for all of these instances...?